python-package/securexgboost/rpc/protos/remote.proto

```diff
@@ -257,3 +263,9 @@ message ClusterParams {
     // Number of nodes in cluster
     uint32 num_workers = 1;  
 }
+
+message RabitParams {
+    string username = 1;
+
+    uint32 status = 2;
```

Comment:
Do we need a `status` in this message?

Reply:
I suppose not functionally, but I added it for completeness

Reply:
Hmm but then other message types should have a similar field too for consistency? Also, `Status status` instead of `uint32`?

Reply:
I replaced `uint32` with `Status`